### ISSUE TITLE
[bug] Device list is not displayed correctly on x64

### ISSUE THREAD

**Issue by jozan** [expo/orbit#35]
### Summary

no iOS simulators show up in menubar dropdown when android sdk is not installed. in pre-flight screen's spinners are spinning forever.

i tried complete reinstall by removing everything with [AppCleaner](https://freemacsoft.net/appcleaner/) and using the new version downloaded from the releases page.

i don't have android dev environment installed at all because of i don't need it right now. this might be the culprit but i don't know. i tried setting the android sdk path to something random but it did not result ios simulators to show up.

i have no idea what would help to debug this in more detail. if you'd like to you can reach me out on expo discord. i use the same nickname there.

### Steps to reproduce

1. open up Expo Orbit and see pre-flight checklist screen
2. see spinners spinning forever trying to resolve something
3. no ios simulators show up in menubar dropdown menu

### Environment

```
  expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 13.5
      Shell: 3.6.1 - /usr/local/bin/fish
    Binaries:
      Node: 20.5.0 - ~/n/bin/node
      Yarn: 3.5.0 - ~/n/bin/yarn
      npm: 9.8.0 - ~/n/bin/npm
      Watchman: 2023.07.24.00 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.12.1 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 22.4, iOS 16.4, macOS 13.3, tvOS 16.4, watchOS 9.4
    IDEs:
      Xcode: 14.3.1/14E300c - /usr/bin/xcodebuild
    npmPackages:
      expo: ^49.0.0 => 49.0.0
      react: 18.2.0 => 18.2.0
      react-native: 0.72.3 => 0.72.3
    npmGlobalPackages:
      eas-cli: 4.1.2
    Expo Workflow: managed
```

`xcrun` shows these things. i have no clue if any of this helps.

```
$ xcrun --show-sdk-path
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk

$ xcrun xcodebuild -version
Xcode 14.3.1
Build version 14E300c

$ xcrun --show-sdk-platform-version
13.3

$ xcrun --show-sdk-version
13.3
```

`xcode-select` shows the correct path where Xcode is installed
```
$ xcode-select -p
/Applications/Xcode.app/Contents/Developer
```

![image](https://github.com/expo/orbit/assets/11707729/7e6b9d3b-462e-4e0d-8e71-af958ef12b14)

**Comment by gabrieldonadel** [comment]
Hi @jozan, thanks for reporting this, can you please double-check that you have version 0.1.1 installed? You can check this by going to the settings windows 

<img width="512" alt="image" src="https://github.com/expo/orbit/assets/11707729/6041a591-7ca2-4f46-8c4d-204922d6f197">


If so can you run this command on your machine and paste the output?
```

**Comment by gabrieldonadel** [comment]
Hi @jozan, thanks for reporting this, can you please double-check that you have version 0.1.1 installed? You can check this by going to the settings windows 

<img width="512" alt="image" src="https://github.com/expo/orbit/assets/11707729/6041a591-7ca2-4f46-8c4d-204922d6f197">


If so, can you run this command on your machine and paste the output?

```sh
# If have a M1/M2 computer
/Applications/Expo\ Orbit.app/Contents/Resources/orbit-cli-arm64 li
st-devices

# If you have an Intel
/Applications/Expo\ Orbit.app/Contents/Resources/orbit-cli-x64 list-devices

```

**Comment by jozan** [comment]
after having a quick debugging session with @gabrieldonadel on discord we concluded that there is, indeed, an issue. the cause is unknown as of now. investigation is ongoing as we speak.

**Comment by gabrieldonadel** [comment]
After the debug session with @jozan, interesting things to note:

- Running `orbit-cli-x64 list-devices` returns an output with a list of devices 
- This is an Intel Mac (not sure if this could be related)
- The Preflight check window was being shown multiple times, (maybe @react-native-async-storage/async-storage is erroring out?)

**Comment by kliakhovskii-brex** [comment]
Potentially also seeing this issue, can't see a list of devices on my intel Mac even though `list-devices` returns sims in them ⬇️ 

![Screenshot 2023-08-11 at 8 16 19 PM](https://github.com/expo/orbit/assets/89865360/b9367e7a-5859-48c4-99d5-61b3fbb5bd42)

Not sure if it's expected, but every time I open `Select build from EAS...` it goes to choosing account / project, and then even if I go into a build and press `Open with Orbit`, it just goes back to opening the menu. I assume it's ok if I had a device in there I could've selected it 🤔

**Comment by gabrieldonadel** [comment]
Ok, I've set up a Hackintosh in an old Intel machine and I'm able to reproduce this, the problem seems related to the communication between `menu-bar` and the `cli`, maybe it's a sandbox configuration that we need to change? I'm installing Xcode in this other machine right and a fix should come soon!

**Comment by gabrieldonadel** [comment]
A new version of Expo Orbit has just been released! [v0.1.2](https://github.com/expo/orbit/releases/tag/expo-orbit-v0.1.2) includes fixes for this and other issues, you can check the full details on the [releases page](https://github.com/expo/orbit/releases). Please test it out and let us know if you find any other problems.
